Few CRIC unit tests failing due to the change of T2_CH_CERN_HLT -> T2_CH_CERN_P5

**Impact of the bug**
WMCore unit tests

**Describe the bug**
As can be seen in Jenkins baseline tests:
https://cmssdt.cern.ch/dmwm-jenkins/job/DMWM-WMCorePy3-UnitTests/

there are 3 CRIC unit tests that started failing. By a quick glance, failure seems to be related to the removal of T2_CH_CERN_HLT site from the catalog (replaced now by T2_CH_CERN_P5).

**How to reproduce it**
Run one of these 3 unit tests:
WMCore_t.Services_t.CRIC_t.CRIC_t.CRICTest.testPNNstoPSNs
WMCore_t.Services_t.CRIC_t.CRIC_t.CRICTest.testPSNtoPNNMap
WMCore_t.Services_t.CRIC_t.CRIC_t.CRICTest.testPNNtoPSNMap

**Expected behavior**
Unit tests are expected to succeed (while properly testing the code, of course :))

**Additional context and error message**
One error message is:
```
======================================================================
FAIL: Test API to get a map of PSNs to PNNs
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/dmwm/wmcore_unittest/WMCore/test/python/WMCore_t/Services_t/CRIC_t/CRIC_t.py", line 277, in testPNNtoPSNMap
    self.assertItemsEqual(result['T2_CH_CERN'], ['T2_CH_CERN_HLT', 'T2_CH_CERN'])
AssertionError: Element counts were not equal:
First has 1, Second has 0:  'T2_CH_CERN_P5'
-------------------- >> begin captured logging << --------------------
root: DEBUG: Service 'CRIC' initialized with the following settings: {'inputdata': {}, 'cacheduration': 1, 'method': None, 'timeout': 300, 'endpoint': 'https://cms-cric.cern.ch/', 'accept_type': 'application/json', 'content_type': 'application/json', 'logger': <RootLogger root (NOTSET)>, 'service_name': 'CRIC', 'requests': {'endpoint': 'https://cms-cric.cern.ch/', 'cacheduration': 1, 'accept_type': 'application/json', 'content_type': 'application/json', 'logger': <RootLogger root (NOTSET)>, 'host': 'https://cms-cric.cern.ch/', 'endpoint_components': ParseResult(scheme='https', netloc='cms-cric.cern.ch', path='/', params='', query='', fragment=''), 'deleteCacheOnExit': <WMCore.Services.Requests.TempDirectory object at 0x400d7c7ca0>, 'cachepath': '/tmp/.wmcore_cache_wisctwdh/cms-cric.cern.ch', 'req_cache_path': '/tmp/.wmcore_cache_wisctwdh/cms-cric.cern.ch/.cache', 'cert': '/home/dmwm/certs/servicecert.pem', 'key': '/home/dmwm/certs/servicekey.pem', 'capath': None, 'timeout': 300}, 'cachepath': '/tmp/.wmcore_cache_wisctwdh/cms-cric.cern.ch'}
root: DEBUG: Initializing CRIC with url: https://cms-cric.cern.ch/
```

--- a/test/python/WMCore_t/Services_t/CRIC_t/CRIC_t.py
+++ b/test/python/WMCore_t/Services_t/CRIC_t/CRIC_t.py
@@ -149,7 +149,7 @@ class CRICTest(EmulatedUnitTestCase):
         self.assertItemsEqual(self.myCRIC.PNNstoPSNs(pnns), ['T1_US_FNAL'])
 
         pnns = ['T2_CH_CERN', 'T2_DE_DESY']
-        psns = ['T2_CH_CERN_HLT', 'T2_DE_DESY', 'T2_CH_CERN']
+        psns = ['T2_CH_CERN_HLT', 'T2_CH_CERN_P5', 'T2_DE_DESY', 'T2_CH_CERN']
         self.assertItemsEqual(self.myCRIC.PNNstoPSNs(pnns), psns)
 
         psns = ['T2_UK_London_IC', 'T3_UK_London_QMUL', 'T3_UK_SGrid_Oxford', 'T3_UK_London_RHUL', 'T3_UK_ScotGrid_GLA']
@@ -226,7 +226,7 @@ class CRICTest(EmulatedUnitTestCase):
 
         # test a exact site name, which is treated as a regex and yields a confusing result!!!
         result = self.myCRIC.PSNtoPNNMap('T2_CH_CERN')
-        self.assertItemsEqual(list(result), ['T2_CH_CERN', 'T2_CH_CERN_HLT'])
+        self.assertItemsEqual(list(result), ['T2_CH_CERN', 'T2_CH_CERN_HLT', 'T2_CH_CERN_P5'])
         self.assertItemsEqual(result['T2_CH_CERN'], ['T2_CH_CERNBOX', 'T2_CH_CERN'])
         self.assertItemsEqual(result['T2_CH_CERN_HLT'], ['T2_CH_CERN'])
 
@@ -274,12 +274,12 @@ class CRICTest(EmulatedUnitTestCase):
 
         result = self.myCRIC.PNNtoPSNMap('T2_CH_CERN$')
         self.assertItemsEqual(list(result), ['T2_CH_CERN'])
-        self.assertItemsEqual(result['T2_CH_CERN'], ['T2_CH_CERN_HLT', 'T2_CH_CERN'])
+        self.assertItemsEqual(result['T2_CH_CERN'], ['T2_CH_CERN_HLT', 'T2_CH_CERN_P5', 'T2_CH_CERN'])
 
         # test a exact site name, which is treated as a regex and yields a confusing result!!!
         result = self.myCRIC.PNNtoPSNMap('T2_CH_CERN')
         self.assertItemsEqual(list(result), ['T2_CH_CERN', 'T2_CH_CERNBOX'])
-        self.assertItemsEqual(result['T2_CH_CERN'], ['T2_CH_CERN_HLT', 'T2_CH_CERN'])
+        self.assertItemsEqual(result['T2_CH_CERN'], ['T2_CH_CERN_HLT', 'T2_CH_CERN_P5', 'T2_CH_CERN'])
         self.assertItemsEqual(result['T2_CH_CERNBOX'], ['T2_CH_CERN'])
 
         result = self.myCRIC.PNNtoPSNMap('T2_CH_CERN_HLT')

